dlp/src/test/java/com/example/dlp/DeIdentificationIT.java

```diff
@@ -56,26 +56,49 @@ public void setUp() {
   @Test
   public void testDeidStringMasksCharacters() throws Exception {
     String text = "\"My SSN is 372819127\"";
-    DeIdentification.main(new String[] {
-        "-m", text,
-        "-maskingCharacter", "x",
-        "-numberToMask", "5"
-    });
+    DeIdentification.main(
+        new String[] {
+          "-m", text,
+          "-maskingCharacter", "x",
+          "-numberToMask", "5"
+        });
     String output = bout.toString();
-    assertEquals(output, "My SSN is xxxxx9127\n");
+    assertThat(output, containsString("My SSN is xxxxx9127"));
   }
 
   @Test
   public void testDeidStringPerformsFpe() throws Exception {
```

Comment:
Would it be worth testing reidentification with FPE as well? (perhaps in a separate test file)